modules/services/window-managers/xmonad.nix

```diff
@@ -150,11 +153,5 @@ in {
         '';
       };
     })
-
-    {
-      home.file = mapAttrs' (name: value:
-        attrsets.nameValuePair (".xmonad/lib/" + name) { source = value; })
-        cfg.libFiles;
-    }
```

Comment:
Should this be moved to `cfg.config == null`?

Reply:
There may be a use-case for providing `home-manager`-managed files in `.xmonad/lib` while having an externally managed xmonad-config, but this code removes these files anyway... so I'm not quite sure I understand the question 😅